features/account/resetting_password_validation.feature

```diff
@@ -15,6 +15,13 @@ Feature: Resetting a password validation
         And I try to reset it
         Then I should be notified that the email is required
 
+    @ui @api
+    Scenario: Trying to reset password with a wrong email
+        When I want to reset password
+        And I specify customer email as "wrongman@example.com"
+        And I try to reset it
+        Then I should be notified that the user email does not exist
```

Comment:
@lchrusciel when executing `@ui` behat tests we are expecting to get a message saying :

```
If the email you have specified exists in our system, we have sent there an instruction on how to reset your password.
```

I guess it's for secuirty reasons, we don't want spammers to know if an email exists or not into our database.

Regarding `@api` behat tests, this false-right message is not displayed and I wonder if we would like to reproduce this behaviour or not in the API.